FlowCrypt/src/test/java/com/flowcrypt/email/api/email/WkdClientTest.kt

```diff
@@ -1,61 +0,0 @@
-/*
- * © 2021-present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com
- * Contributors:
- *   Ivan Pizhenko
- */
-
-package com.flowcrypt.email.api.email
-
-import com.flowcrypt.email.api.wkd.WkdClient
```

Comment:
why is this completely removed? what is instead of it?

Reply:
After refactoring `WkdClient` requires `Context`(Android things). For now, I can't provide `Context` for JUnit tests. But maybe I'll find out how to do that and restore this test. Anyway, most of them are covered by [CreateMessageActivityWkdTest](https://github.com/FlowCrypt/flowcrypt-android/blob/7ba8e4412def7b33035975cf657b02b315700d43/FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/activity/CreateMessageActivityWkdTest.kt)

Reply:
Could it be left untouched and wrapped into Context in a different class?

Reply:
Unfortunately, I can't leave it as-is because it is not optimized for Android(retrofit, coroutines, and so on)

Reply:
@IvanPizhenko I've restored this test with a few changes. Timeout case is covered by Android Tests.